.circleci/deploy-python-3.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+set -e
+
+VERSION_CHANGE_TRIGGERS="setup.py MANIFEST.in openfisca_core openfisca_web_api_preview"
```

Comment:
@MattiSG I'm not sure about repeating this in 3 different files, as the whole continuous publishing relies on this variable to have the same values in the 3 files. 

However, I didn't manage to share it. I tried to declare it in a separate shell file and to source it from each of the 3 scripts, but the `source` would not find the file (I think because relative paths in a script are always relative to where you launch the script, not where the script is).

If you know how to make it better, please proceed 🙂

Reply:
I'll just extract that in a new script :wink:

If you're curious, [this is how](https://github.com/MattiSG/DotFiles/blob/1e33187131229a23aa2467f6f54e44d800c3a470/source/prompt.sh#L1) I usually solve the problem.